### DISCUSSION TITLE
WED files attribute "Unused bytes?"

### DISCUSSION THREAD

**Discussion by Incrementis** [NearInfinityBrowser/NearInfinity#183] [Q&A] [answered]
When manually searching the WED files, two of them can be found in which the attribute “_Unused Bytes?_” is present. 
These are:

- **AR1000.WED**
![grafik](https://github.com/user-attachments/assets/82a90e83-5334-4614-9cb5-f82948cf1181)

- **OH2010.WED**
![grafik](https://github.com/user-attachments/assets/fa6a9150-8b1d-46f9-b77b-fdbb3e4063ae)

I think they don't cause any problems since I tested **AR1000.WED** rudimentarily with no errors in the game (but I could be wrong). Interestingly, both “_Unused Bytes?_” are different sizes (see screenshots).

A superficial explanation I have is that it's a door(s), so theoretically it can be any size?
Another question is why “only” these two WED files have this attribute (What causes “_Unused Bytes?_” and can it be prevented)?

**Comment by Argent77** [comment, marked as answer]
NI marks regions of unassigned data  as `Unused bytes?`. The most likely reasons are either incorrect header definitions or bogus data. Finding the true cause can be difficult since the WED file format is rather complicated. Could be a potential candidate for the EE Fixpack.

**Comment by Incrementis** [comment]
Thanks for the reply. I'll point this conversation to the EE Fixpack GitHub repo. Lets see how it goes on.
